repologyapp/views/graphs_map.py

```diff
@@ -128,11 +128,17 @@ def point_coords(point: _MapPoint) -> Tuple[float, float]:
             int(height - offset_bottom - point.y / max_y * (height - offset_top - offset_bottom)) + 0.5
         )
 
-    with doc.tag('g', ('font-family', 'DejaVu Sans,Verdana,Geneva,sans-serif'), ('font-size', 11), ('text-anchor', 'start')):
-        for point in points:
-            if point.text:
-                x, y = point_coords(point)
-                with doc.tag('text', ('stroke-linecap', 'round'), ('stroke-width', 3), fill='#f0f0f0', stroke='#f0f0f0', x=x + 5, y=y + 3):
+    for point in points:
+        if point.text:
+            x, y = point_coords(point)
+            if x / max_x >= 0.9:
```

Comment:
The condition is inverted and also you need to use `width` here, as `max_x` is maximal data value (e.g. max # of packages), while you are dealing with pixels here.